src/compat.jl

```diff
@@ -3,6 +3,16 @@ This file defines internal functions for compatibility across
 different Julia versions.
 =#
 
+using Compat
+import Compat.String
+
+if VERSION < v"0.7-"
+    import Base.LinAlg:norm, checksquare
+else
+    using SparseArrays, LinearAlgebra
```

Comment:
Out of curiosity: What is `SparseArrays` used for?

Reply:
for example in julia 0.7 `sparse` or `sparsevec` needs to do `using SparseArrays`.

but i can avoid having to write these `using` each time the REPL is started with the startup script that should be in `.julia/config`: 

```
$ pwd
/Users/forets/.julia
$ cat config/startup.jl
using LinearAlgebra,
      Pkg,
      SparseArrays,
      Test,
      BenchmarkTools
```

Reply:
No, it's fine!